tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -195,7 +195,7 @@ public:
    REntry *GetDefaultEntry() const;
 
    RFieldZero *GetFieldZero() const { return fFieldZero.get(); }
-   const Detail::RFieldBase *GetField(std::string_view fieldName) const;
+   Detail::RFieldBase *GetField(std::string_view fieldName) const;
```

Comment:
As per discussion in #10408, can this go back to `const`?

Reply:
done